CustomCraftSML/Interfaces/IAliasRecipe.cs

```diff
@@ -1,6 +1,6 @@
 ﻿namespace CustomCraft2SML.Interfaces
 {
-    public interface IAliasRecipe : IAddedRecipe
+    internal interface IAliasRecipe : IAddedRecipe
```

Comment:
Reorganize the interfaces into two new categorizations:  
1. Interfaces that define the EasyMarkup structure
2. Interfaces only used internally (consider if a new base class would be useful)